app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
+	
+Here's an example:
+
+[See it on Plunker](http://plnkr.co/edit/VPktVlI0PWMnYdYVsVBx?p=preview)
+
+```html
+<!-- Create a template for the custom element -->
+<dom-module id='custom-element'>
+  <template>
+    <h1>Heading!</h1>
+    <p>We are elements in custom-element's local DOM.</p>
+  </template>
+</dom-module>
+
+<!-- Drop the custom element on the page -->
```

Comment:
So if I add 

`<script src="webcomponentsjs/webcomponents-loader.js"></script>`

after the `<link rel=blablablahb>` to every index.html, would that work?